src/client-api-proxy.ts

```diff
@@ -22,6 +23,22 @@ export async function getStories(
 ): Promise<BrazeContentProxyResponse> {
   let data: ClientApiResponse | null = null;
 
+  // Validate inputs
+  try {
+    await validateScheduledSurfaceGuid(scheduledSurfaceID);
+  } catch (err) {
+    console.log(err);
+    Sentry.captureException(err);
+  }
+
+  try {
+    await validateDate(date);
+  } catch (err) {
+    console.log(err);
+    Sentry.captureException(err);
+  }
+
+  // Retrieve data
   try {
```

Comment:
can use the single try-catch block here, and do all validations within this try block.